package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "lint": "./node_modules/.bin/standard --verbose src/**",
-    "test": "rm -rf ./tmp ; jest --forceExit --coverage --runInBand --testURL=\"http://localhost\"",
+    "test": "rm -rf ./tmp && jest --forceExit --coverage --runInBand --testURL=\"http://localhost\"",
```

Comment:
I thought `;` and `&&` are equivalent?

Reply:
yeah basically, && just only runs if first command doesn't fail, just wanted to make things actually deleted, thought i had some test issues due to that

Reply:
I see, great!